tests/app/dao/test_fact_billing_dao.py

```diff
@@ -507,7 +510,7 @@ def test_fetch_usage_for_service_by_month_variable_rates(notify_db_session):
 
 
 @freeze_time('2018-08-01 13:30:00')
-def test_fetch_usage_for_service_by_month_adds_data_for_today(notify_db_session):
+def test_fetch_usage_for_service_by_month_populates_ft_billing_for_today(notify_db_session):
```

Comment:
Yes, it's something that could still be worthwhile. It wasn't a critical part of supporting variable rates so I haven't tried it here. When I did look at it I struggled to think of what the separate files would be called, which made me wonder if it's really a good approach - it may be the names don't matter, but it did cause me to pause.

There's still a fair bit of DRYing-up and refactoring we can do in the main code and here. We may find that once this is done the files are both of a reasonable size. (As discussed, these refactoring actions are documented on the card.)